keps/2724-topology-aware-schedling/README.md

```diff
@@ -344,8 +344,9 @@ type ResourceFlavorSpec struct {
 
 // TopologySpec defines the desired state of Topology
 type TopologySpec struct {
-  // Levels defines the levels of topology.
+  // levels define the levels of topology.
   //
+  // +required
   // +listType=atomic
   // +kubebuilder:validation:MinItems=1
   // +kubebuilder:validation:MaxItems=5
```

Comment:
Let's be a bit more generous here and bump it to 8, so that it covers all possible use cases, and and the same time has a sane max limit.

Reply:
sure, done